extensions/positron-r/src/r-installation.ts

```diff
@@ -48,7 +53,7 @@ export enum ReasonDiscovered {
 	HQ = "HQ",
 	/* eslint-enable @typescript-eslint/naming-convention */
 	adHoc = "adHoc",
-	user = "user",
+	userSetting = "userSetting",
```

Comment:
I've got this building sense that we should just be saying "setting" instead of "user" (already underway) or "userSetting", because I think most of the recent and ongoing work around this is actually for the workbench context and it's really "adminSetting" most of the time. This is not (necessarily) a concrete ask for this PR, but I wanted to express this. And it's conceivable it has implications for this PR.

This also makes me think about `git config --list --show-origin` and how nice it is for understanding which config file (system, user, repo) is responsible for a particular setting. That's yet another level of not conflating all of this or suggesting that it's all happening at the user level.

Reply:
Melissa and I also discussed the naming here: https://github.com/posit-dev/positron/pull/6621#discussion_r1980175178

@melissa-barca do you feel strongly about `userSetting` vs `setting`?

Regardless of if the setting is configured by a user or a Workbench Admin, the setting is configured via User Settings (admin workflow involves merging configured settings into the user's settings). I think we are likely to document these settings as "user settings" even in the Workbench Admin docs?

Reply:
I do think User Setting will be more clear than Setting. @sharon-wang  is documenting this in the Workbench Admin Guide as user settings. It does bring up a good point that we should call out this description and the potential for the admin to have configured defaults in the Workbench User Guide. Most likely here: https://docs.posit.co/ide/server-pro/user/positron/guide/settings.html#user-settings

Sharon, do you want me to add that request to your existing workbench doc issue or file a new issue for the Workbench team?

Reply:
Ok, I'll keep `userSetting` as-is for now.

I've added a couple extra notes to my existing workbench doc issue to cover those in my docs PR!